src/Module/Conversation/Network.php

```diff
@@ -148,6 +148,64 @@ public function __construct(
 		$this->userDefinedChannel = $userDefinedChannel;
 	}
 
+	protected function widgets(string $widget)
```

Comment:
I'd prefer

```suggestion
	protected function gerWidgetContent(string $widget): string
```
 And return the translated widget string and add it to the `aside` array-entry as it makes it better readable I guess

Reply:
Right now it's no longer a method call so this can't really be done anymore. Do we want the method back?